examples/DotNetCore/WebDemo/Microsoft.Azure.AppConfiguration.WebDemo/appsettings.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "Logging": {
+    "IncludeScopes": false,
+    "LogLevel": {
+      "Default": "Warning"
+    }
+  },
+
+  "Settings": {
+    "AppName": "Azure App Configuration Demo",
```

Comment:
Write now the code won't load this because the code is using "WebDemo:Settings:Messages" etc.

Let's change this section to 
```
//
// Uncomment section to use local app settings
//
/*
  "WebDemo": {
    "Settings": {
      "AppName": "Azure App Configuration Demo",
      "Version": 1.0,
      "FontSize": 50,
      "RefreshRate": 1000,
      "Language": "English",
      "Messages": "Hello There;Thanks For Using Azure App Configuration",
      "BackgroundColor": "Orange"
    }
  }
*/
```

Reply:
Added settings for local usage.